website/e2e-test/README.md

```diff
@@ -0,0 +1,19 @@
+## Installing dependencies
```

Comment:
We have an open issue to convert all distributed READMEs to more brief explanations of what the current directory is for, and point to a relevant doc in the docs/ directory.

Approving this, but also a reminder that we're trying to centralize the key information.

Reply:
Moved this into `docs/tutorials`. 👍